sdk/communication/Azure.Communication.Common/src/EntraTokenGuardPolicy.cs

```diff
@@ -46,5 +46,34 @@ private async ValueTask ProcessAsync(HttpMessage message, ReadOnlyMemory<HttpPip
                 return;
             }
         }
+
+        private (bool CacheOutdated, string EntraToken) IsEntraTokenCacheOutdated(HttpMessage message)
```

Comment:
It reads better to use positive language, i.e. `IsEntraTokenCacheValid` and `CacheValid` and only negate in the conditions as needed.